portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -315,6 +315,8 @@ definition {
 
 		Portlet.addPG(portletName = "Documents and Media");
 
+		Navigator.gotoPage(pageName = "Test Page");
```

Comment:
Why not try to simply wait for `D&M` to be loaded instead of doing a navigation? Wasn't this triggering the `portletAdded` or some other event we were tracking?

Reply:
Good point, but when looking at what's already run within the test, the macro to add a portlet already contains the waitForPortletReady event. So it must be something else that's causing the poshi drag and drop function to trip in this case...

-  [Fails] - Without the page navigation, already has waitfor portletReady event listener:
![image](https://user-images.githubusercontent.com/35235266/115054151-c147f900-9e94-11eb-9bd7-7ccd8e034adb.png)
[Poshi Report](https://testray.liferay.com/reports/production/logs/2021-04/test-1-8/test-portal-testsuite-upstream(master)/734/functional-websphere90-mysql57-jdk8/1/10/LocalFile.UIInfrastructureUsecase_ViewPortletBoundaries/index.html.gz)

- [Passes] With the page navigation:
![image](https://user-images.githubusercontent.com/35235266/115053729-4bdc2880-9e94-11eb-9356-cd17d8aef616.png)
[Poshi Report](https://testray.liferay.com/reports/production/logs/2021-04/test-1-1/test-portal-acceptance-pullrequest%28master%29/6601/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.UIInfrastructureUsecase_ViewPortletBoundaries/index.html.gz)